Plugin/Model/PagePlugin.php

```diff
@@ -48,8 +56,22 @@ public function aroundGetContent(Page $subject, callable $proceed)
             return $proceed();
         }
 
-        $this->remotePageContentCache[$remotePageId] = $remotePage['content']['rendered'];
+        $html = $remotePage['content']['rendered'];
+
+        $html = preg_replace_callback("{{(.*)}}", function ($matches) {
+            $match = $matches[0];
+
+            $match = html_entity_decode($match);
+            $match = str_replace("”", "\"", $match); // Opening quote
```

Comment:
Why not use single `'` quotes? Seems more readable and it is standard compliant.

Reply:
Good point, will change.